ignored_security_issues.json

```diff
@@ -0,0 +1,3 @@
+[
```

Comment:
Also, this will need to be updated to include a reason. This issue has this reason:
```json
[
    {
        "vulnerability_id": "",
        "reason": "This is a false positive, see https://github.com/github/advisory-database/issues/3280"
    }
]
````

Reply:
The reason note has been added.